### DISCUSSION TITLE
EssentialsX problem with Featherboard

### DISCUSSION THREAD

**Discussion by Guest1131** [EssentialsX/Essentials#4163] [Q&A] [answered]
Hi, I have a problem with the latest build of essentialsX and the Featherboard plugin.

https://www.spigotmc.org/resources/featherboard.2691/

When featherboard is installed on the server, essentialsX gets deactivated and turns red in the plugins /plugins list.

I tried to update featherboard, but nothing changes. If I add an older version of EssentialsX, everything works fine.

Use Spigot 1.8.8

Can you fix this problem? Please

Thanks

**Comment by JRoy** [comment]
We need a startup log to help you.

**Reply by Guest1131** [reply]
https://pastebin.com/G8vLCtKW

**Reply by Guest1131** [reply]
I can't figure out if it's a problem of essentials or Featherboard

**Reply by mdcfe** [reply]
Related: #4162

**Comment by triagonal** [comment, marked as answer]
Resolved in #4166.
